infrastructure/main.tf

```diff
@@ -7,125 +7,125 @@ locals {
 
   preview_app_service_plan     = "${var.product}-${var.component}-${var.env}"
   non_preview_app_service_plan = "${var.product}-${var.env}"
-  app_service_plan             = "${var.env == "preview" || var.env == "spreview" ? local.preview_app_service_plan : local.non_preview_app_service_plan}"
+  app_service_plan             = var.env == "preview" || var.env == "spreview" ? local.preview_app_service_plan : local.non_preview_app_service_plan
 
   preview_vault_name           = "${var.raw_product}-aat"
   non_preview_vault_name       = "${var.raw_product}-${var.env}"
-  key_vault_name               = "${var.env == "preview" || var.env == "spreview" ? local.preview_vault_name : local.non_preview_vault_name}"
+  key_vault_name               = var.env == "preview" || var.env == "spreview" ? local.preview_vault_name : local.non_preview_vault_name
 
 }
 
 resource "azurerm_resource_group" "rg" {
   name     = "${var.product}-${var.component}-${var.env}"
-  location = "${var.location}"
-  tags     = "${merge(var.common_tags, map("lastUpdated", "${timestamp()}"))}"
+  location = var.location
+  tags     = merge(var.common_tags, map("lastUpdated", "${timestamp()}"))
 }
 
 data "azurerm_key_vault" "ia_key_vault" {
-  name                = "${local.key_vault_name}"
-  resource_group_name = "${local.key_vault_name}"
+  name                = local.key_vault_name
+  resource_group_name = local.key_vault_name
 }
 
 data "azurerm_key_vault_secret" "appeal_submission_template" {
   name      = "appeal-submission-template"
-  vault_uri = "${data.azurerm_key_vault.ia_key_vault.vault_uri}"
+  vault_uri = data.azurerm_key_vault.ia_key_vault.vault_uri
 }
 
 data "azurerm_key_vault_secret" "test_caseofficer_username" {
   name      = "test-caseofficer-username"
-  vault_uri = "${data.azurerm_key_vault.ia_key_vault.vault_uri}"
+  vault_uri = data.azurerm_key_vault.ia_key_vault.vault_uri
 }
 
 data "azurerm_key_vault_secret" "test_caseofficer_password" {
   name      = "test-caseofficer-password"
-  vault_uri = "${data.azurerm_key_vault.ia_key_vault.vault_uri}"
+  vault_uri = data.azurerm_key_vault.ia_key_vault.vault_uri
 }
 
 data "azurerm_key_vault_secret" "test_law_firm_a_username" {
   name      = "test-law-firm-a-username"
-  vault_uri = "${data.azurerm_key_vault.ia_key_vault.vault_uri}"
+  vault_uri = data.azurerm_key_vault.ia_key_vault.vault_uri
 }
 
 data "azurerm_key_vault_secret" "test_law_firm_a_password" {
   name      = "test-law-firm-a-password"
-  vault_uri = "${data.azurerm_key_vault.ia_key_vault.vault_uri}"
+  vault_uri = data.azurerm_key_vault.ia_key_vault.vault_uri
```

Comment:
the changes in this file won't work, you aren't on the terraform version required for this

Reply:
Tim did